images/custom-error-pages/rootfs/main.go

```diff
@@ -97,6 +97,19 @@ func main() {
 	http.ListenAndServe(fmt.Sprintf(":8080"), nil)
 }
 
+func findAcceptedFormat(requestedMimes string) (string, error) {
+	// get request format from Accept header
+	for _, requestedMime := range strings.Split(requestedMimes, ",") {
+		requestedMime = strings.TrimSpace(requestedMime)
+		exts, err := mime.ExtensionsByType(requestedMime)
+		if err != nil || len(exts) == 0 {
+			continue
+		}
+		return exts[0], nil
```

Comment:
This var naming is a bit confusing. I understand each var is on its own scope, but let not call all of them as "requestedMime" and give a meaningful name for them.

Reply:
I did some updates, please have a look.